README.md

```diff
@@ -26,8 +26,7 @@ Options for fmt:
 
 Options for sync:
   -e, --exercise <slug>        Only operate on this exercise
-  -p, --prob-specs-dir <dir>   Use this 'problem-specifications' directory, rather than cloning temporarily
-  -o, --offline                Do not check that the directory specified by --prob-specs-dir is up to date
+  -o, --offline                Do not check that the cached 'problem-specifications' directory is up to date
```

Comment:
I wonder if we could shorten this a bit and make it describe the functionality more than the implementation. 
Maybe something like:

```suggestion
  -o, --offline                Do not update the 'problem-specifications' information
```

or 

```suggestion
  -o, --offline                Do not update the 'problem-specifications' data
```

or

```suggestion
  -o, --offline                Do not update the 'problem-specifications' repo
```

Reply:
Good idea, thanks. The previous version was also bad because it only reflected the old behavior of "is it out of date?" rather than "pull from prob-specs".

I pushed this:

```
  -o, --offline                Do not update the cached 'problem-specifications' data
```

where maybe the word "cached" helps convey that `--offline` won't work if you don't have a local prob-specs.

But I don't like how "update" looks next to the rest of the help text:

```
Options for sync:
  -e, --exercise <slug>        Only operate on this exercise
  -o, --offline                Do not update the cached 'problem-specifications' data
  -u, --update                 Prompt to update the seen data that are unsynced
  -y, --yes                    Auto-confirm prompts from --update for updating docs, filepaths, and metadata
      --docs                   Sync Practice Exercise '.docs/introduction.md' and '.docs/instructions.md' files
      --filepaths              Populate empty 'files' values in Concept/Practice exercise '.meta/config.json' files
      --metadata               Sync Practice Exercise '.meta/config.json' metadata values
      --tests [mode]           Sync Practice Exercise '.meta/tests.toml' files.
                               The mode value specifies how missing tests are handled when using --update.
                               Allowed values: c[hoose], i[nclude], e[xclude] (default: choose)
```

where the `--update` text of "prompt to update the seen data" is now confusing - it might imply that prob-specs isn't updated unless `--update` is passed, which is wrong.

I've pushed a commit that changes the `--update` text to "seen track data", but it still doesn't seem great. Maybe we could avoid the word "update" in `--offline`, or the word "update" in `--update`? Thoughts?

One more extreme option is to change `--update` to `--write`...